flexmeasures/api/v1_1/tests/conftest.py

```diff
@@ -39,14 +39,14 @@ def setup_api_test_data(db, setup_accounts, setup_roles_users, add_market_prices
     for asset_name in asset_names:
         asset = Asset(
             name=asset_name,
+            owner_id=test_user.id,
```

Comment:
I assume you are adding this now, so that the GenericAsset is set up with the correct account ID?

Maybe a stupid question, but is this not a good time to set up the tests with `GenericAsset` / `Sensor` already (seeing that you test for `Sensor` now)?

Reply:
Your assumption is correct.

Updating tests (to skip creating the Asset in favour of directly creating the Sensor) comes later. As long as there is still code that depends on the Asset being there, we need to create both the Asset and the corresponding Sensor. Your comment did prompt me to add this task as a note to https://github.com/SeitaBV/flexmeasures/projects/11, right before removing the old sensor classes.